src/Log/LogService.php

```diff
@@ -291,7 +285,7 @@ public function setDefaultPriority($priority)
         } else if (is_string($priority)) {
             $this->defaultPriority = $this->convertPriorityFromString($priority);
         } else {
-            throw new Exception('Setting default priority with invalid parameter.');
+            throw new \Exception('Setting default priority with invalid parameter.');
```

Comment:
No actually not. I am using `Opus\Exception`. The change cause \Exception to be used.